v2/thirdparty/common-customizations/saml/with-boxyhq/integration-steps.mdx

```diff
@@ -28,7 +36,76 @@ You can learn more about these in the [SAML Jackson docs](https://boxyhq.com/doc
 
 In our example app, we use [mocksaml.com](https://mocksaml.com/) which is a free SAML provider that we can use for testing. When you navigate to the site, you will see a "Download metadata" button which you should click on to get the `.xml` file.
 
-## 2) Convert the `.xml` file to base64
+### 2) Start the SAML Jackson service
+
+You can run SAML Jackson [with or without Docker](https://boxyhq.com/docs/jackson/deploy/service).
+```bash
+docker run \
+  -p 5225:5225 \
+  -e JACKSON_API_KEYS="secret" \
+  -e DB_ENGINE="sql" \
+  -e DB_TYPE="postgres" \
+  -e DB_URL="postgres://postgres:postgres@postgres:5432/postgres" \
+  -d boxyhq/jackson
+```
+
+Instead, if you are following the example guide, run:
+
+```bash
+yarn dev
+```
```

Comment:
huh? What example, and why just yarn dev? Keep things simple.. 

Also mention for self hosted, you need to run boxy yourself, but for managed service, you can request us to give you an instance

Reply:
This is also try in the section below via the api

Reply:
Done